src/main/resources/site/mixins/form-intermediate-step-data/form-intermediate-step-data.xml

```diff
@@ -0,0 +1,70 @@
+<mixin>
+    <form>
+        <field-set>
+            <label>Mellomsteg</label>
+            <items>
+                <input name="editorial" type="HtmlArea">
```

Comment:
Bør legge til dette feltet i [htmlarea-utils](https://github.com/navikt/nav-enonicxp/blob/ee70475a4599b589c778eabccfcbb11bb71378d6/src/main/resources/lib/utils/htmlarea-utils.ts#L4), slik at referanse-søk for cache-invalidering ol fungerer som det skal.